core/src/test/java/greencity/service/impl/PlaceCommentServiceImplTest.java

```diff
@@ -0,0 +1,96 @@
+package greencity.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
+
+import greencity.ModelUtils;
+import greencity.dto.PageableDto;
+import greencity.dto.comment.AddCommentDto;
+import greencity.dto.comment.CommentAdminDto;
+import greencity.dto.comment.CommentReturnDto;
+import greencity.entity.Comment;
+import greencity.repository.PlaceCommentRepo;
+import greencity.service.PlaceService;
+import greencity.service.UserService;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import org.junit.Test;
```

Comment:
Annotations reside in the org.junit.jupiter.api package instead of org.junit package.